### PR TITLE
fix: Support USNat and USSec

### PR DESCRIPTION
**Pull request by AbelHu**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/helpers/helpers.go [hunk 1]
```diff
@@ -211,6 +211,8 @@ func GetCloudTargetEnv(location string) string {
 		return "AzureGermanCloud"
 	case strings.HasPrefix(loc, "usgov") || strings.HasPrefix(loc, "usdod"):
 		return "AzureUSGovernmentCloud"
+	case strings.HasPrefix(strings.ToLower(loc), "usnat") || strings.HasPrefix(strings.ToLower(loc), "ussec"):
```
Comment (andyzhangx) [inline pkg/helpers/helpers.go:214]: cc @xizhamsft

Comment (AbelHu) [reply pkg/helpers/helpers.go:214]: I will prepare the PR for aks and ping @xizhamsft

Comment (xuto2) [reply pkg/helpers/helpers.go:214]: No please don't have this here.

Comment (AbelHu) [reply pkg/helpers/helpers.go:214]: Removed. I thought it does not matter. you can find the location in the list https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet

**Pull request merged**